easybuild/easyconfigs/t/tbl2asn/tbl2asn-02272018-linux64.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'CmdCp'
+
+name = 'tbl2asn'
+version = '02272018'
+versionsuffix = '-linux64'
+
+homepage = 'https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/'
+description = """Tbl2asn is a command-line program that automates the creation of
+ sequence records for submission to GenBank"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['ftp://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/tbl2asn/']
+sources = ['linux64.tbl2asn.gz']
```

Comment:
@pescobar Please let EasyBuild rename on download to somehow include `version` in the saved tarball?